packages/block-editor/src/components/block-list/style.scss

```diff
@@ -512,6 +512,25 @@
 		border-right-color: $gray-900;
 	}
 
+	&.is-fixed {
+		position: sticky;
```

Comment:
I feel the is-fixed styles and logic are misplaced, they make more sense  in the `edit-*` packages because we don't really know the container here and these styles depend on their container right?

Reply:
Sure, we could move it. I don't mind either way. `sticky` just falls back to being relative if there's no scroll container though.

Reply:
For example, in the site editor there's no scroll container, but the content box below is scrollable.